packages/mds-compliance-engine/tests/engine/engine-test.spec.ts

```diff
@@ -146,7 +151,51 @@ describe('Verifies compliance engine processes by vehicle most recent event', ()
     await cache.seed({ devices, events, telemetry: [] })
     await Promise.all(devices.map(async device => db.writeDevice(device)))
     const inputs = await getAllInputs()
-    const complianceResults = await processPolicy(LOW_COUNT_POLICY, geographies, inputs)
+    const complianceResults = processPolicy(ARBITRARY_EVENT_TYPES_POLICY, geographies, inputs)
+    const { 0: result } = complianceResults.filter(
```

Comment:
Sidenote, a more fluid syntax (IMO at least) for getting the first element from a list is `const [first] = someList`